notebooks/JSON RPC API.ipynb

```diff
@@ -344,7 +344,18 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "solana_client.get_account_info(PublicKey(1))"
```

Comment:
Please try out the `dataSlice` param here.

Reply:
Added tests.